brian2/tests/test_spatialneuron.py

```diff
@@ -745,6 +745,44 @@ def test_spatialneuron_morphology_assignment():
     assert_allclose(neuron.sec1.sec11.v[:], np.ones(2)*volt)
     assert_allclose(neuron.sec1.sec12.v[:], np.zeros(2)*volt)
 
+@attr('codegen-independent')
```

Comment:
The test is good, but currently it is actually only tested for the `numpy` target because it is marked as `codegen-independent` (which is meant for tests that check things at the description level and don't do a run). It should be  marked `@attr('standalone-compatible', 'multiple-runs')` and include an explicit `device.build` call (as described [here](http://brian2.readthedocs.io/en/latest/developer/guidelines/testing.html#unit-tests))